marimbabot_ur5_moveit_config/.setup_assistant

```diff
@@ -0,0 +1,11 @@
+moveit_setup_assistant_config:
+  URDF:
+    package: marimbabot_description
+    relative_path: urdf/marimbabot_ur5.urdf.xacro
+    xacro_args: ""
```

Comment:
we can remove xacro_args since it's empty stirng

Reply:
I think it is a nessesary argument even if it is empty